crates/ethereum-rpc/src/lib.rs

```diff
@@ -730,7 +715,7 @@ fn register_rpc_methods<C: sov_modules_api::Context, Da: DaService>(
                     .send_raw_tx(data)
                     .await;
 
-                tx_hash.map_err(|e| to_jsonrpsee_error_object(e, ETH_RPC_ERROR))
+                tx_hash.map_err(|e| to_jsonrpsee_error_object(&e.to_string(), e))
```

Comment:
I believe we return PoolError from sequencer

which can be converted to EthApiError

which impls. into JsonRPSeeError 

so proper casting here make it work very well.